test/FakeStateMachine-run.ts

```diff
@@ -123,4 +124,60 @@ describe('FakeStateMachine#run()', () => {
       );
     });
   });
+
+  describe('when Pass state has undefined ResultPath', () => {
+    test('should not throw an Error', async () => {
+      const definition = require('./fixtures/definitions/pass-state-undefined-resultpath.json');
+      const fakeStateMachine = new FakeStateMachine(definition, {});
+
+      expect(
+        await fakeStateMachine.run({
+          changed: 'TBD',
+          unchanged: 'unchanged',
+        })
+      ).toEqual(
+        new RunStateResult(
+          {
+            changed: 'changed by pass 0',
+            unchanged: 'unchanged',
+          },
+          'Succeed',
+          null,
+          true
+        )
+      );
+    });
+  });
+
+  describe('should have execution path', () => {
+    const definition = require('./fixtures/definitions/execution-path-multiple.json');
+
+    test('execution path should be valid', async () => {
+      const fakeStateMachine = new FakeStateMachine(definition, {});
+
+      await fakeStateMachine.run({});
+
+      expect(fakeStateMachine.executionPath).toEqual([
+        'Step1',
+        'Step2',
+        'Choice3',
+        'Rule2',
+        'Success',
+      ]);
+    });
+
+    test('execution path should be invalid', async () => {
+      const fakeStateMachine = new FakeStateMachine(definition, {});
+
+      await fakeStateMachine.run({});
+
+      expect(fakeStateMachine.executionPath).not.toEqual([
```

Comment:
Could you describe an expected behaviour? I think `fakeStateMachine.executionPath` should be empty list, wdyt?

Reply:
This is validating that the `executionPath` does not contain evaluated rules for choice states. The exected behavior should only have executed states in the order of execution. I'm checking this by making sure that the state `NonExecutingRule` has not been added to the `executionPath`.

Reply:
I still don't fully understand

The assertion at L136 seems to be more severe, so we can simply remove this test case, can't we?

Reply:
Sure, I'll remove it now.